### PR TITLE
Add lru for eth_call

### PR DESCRIPTION
**Pull request by giskook**
Closes #<issue number>.

### Reviewers

Main reviewers:

- @John
- @Doe

Codeowner reviewers:

- @-Alice
- @-Bob

### CODE REVIEW

File: jsonrpc/lru_xlayer/lru_xlayer.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+func (l *LRUCache) Set(key string, value interface{}) error {
+	if l == nil || !GetConfig().Enable {
+		return nil
+	}
+	l.cache.Add(key, value)
+	return nil
+}
+
+// GetLRU returns the LRU cache instance.
+func GetLRU() *LRUCache {
```
Comment (zibuyu28) [inline jsonrpc/lru_xlayer/lru_xlayer.go:48]: maybe cause nil panic

Comment (giskook) [reply jsonrpc/lru_xlayer/lru_xlayer.go:48]: No idea about the nil panic

**Review by zjg555543 — APPROVED** [review]

**Review by AungThuSoe24 — COMMENTED** [review]
Withdrawal USDC to another Wallet

**Review by AungThuSoe24 — COMMENTED** [review]
Withdrawal

**Pull request closed without merging**